src/main/java/com/celements/rights/access/DefaultRightsAccessFacade.java

```diff
@@ -79,4 +79,13 @@ public boolean hasAccessLevel(EntityReference ref, EAccessLevel level, XWikiUser
         user, ret);
     return ret;
   }
+
+  public boolean isLoggedIn() {
+    if (!"XWiki.XWikiGuest".equals(getContext().getUser())) {
+      return true;
```

Comment:
for clarity reasons, it's preferable to only have one return statement at the end of the method, e.g.:
```
boolean ret = ...;
LOGGER...
return ret;